aiohttp/client_proto.py

```diff
@@ -10,7 +10,7 @@
 class ResponseHandler(DataQueue, asyncio.streams.FlowControlMixin):
     """Helper class to adapt between Protocol and StreamReader."""
 
-    def __init__(self, *, loop=None, **kwargs):
```

Comment:
Why?

Reply:
kwargs were not being used at this `__init__`